### DISCUSSION TITLE
Can I use booster on fedora?

### DISCUSSION THREAD

**Discussion by arun54321** [anatol/booster#136] [Q&A] [answered]
Is it supposed to be used only on arch and void Linux?

**Comment by anatol** [comment, marked as answer]
Booster aims to be a multi-distribution multi-platform initramfs solution. It is currently in the official repos of Arch Void and Alpine Linux distros but we are also looking for help with adopting project for distros like Debian, Fedora,  CentOS. It would be also great to see booster at non-Linux platforms such as FreeBSD, OpenWrt etc..

I personally do not use Fedora and have almost zero experience with it. But if you or anyone else is interested in creating a package for Fedora I will be glad to help answer questions or provide source patches if needed.

**Reply by arun54321** [reply]
Installed it on fedora. It is working fine. Thanks.

**Reply by anatol** [reply]
Would it be possible to create a package for fedora and put it into the official repo? This would make  installation easier for other people.

**Reply by arun54321** [reply]
Need to write install script ~~for grub~~ to handle kernel updates. Fedora has script only for dracut.

1. 50-dracut.install
2. 51-dracut-rescue.install
